Week-5/symbols/script.js

```diff
@@ -0,0 +1,22 @@
+// We are using an IIFE (Immediately Invoked Function Expression)
+// to avoid polluting the global scope with our counter variable
+const generateTransactionId = (function() {
+    let counter = 0; // This will be private to the generateTransactionId function
+  
+    return function() {
+      counter += 1; // Increment the counter for each ID generated
+      // Use a template string to generate a unique part of the symbol
+      return Symbol(`TRANSACTION_ID_${counter}`);
+    };
+  })();
+  
+  // Example usage:
```

Comment:
The variable names like transaction1, transaction2, and transaction3 are clear and descriptive. Consider adding a use case where the uniqueness of Symbols is critical, such as property keys in an object, to further illustrate the concept.